### ISSUE TITLE
The `complexity` rule shows "anonymous" for shorthand methods.

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#3955]
The rule should show the method name instead of "anonymous".
Plus, the position should be at the head of each method name.

**Code:**

```js
/* eslint-env es6 */
/* eslint complexity: [2, 1] */
var obj = {
    foo(a) {
        return a ? 0 : 1;
    }
};
```

**Actual:**

ESLint v1.5.1

![image](https://cloud.githubusercontent.com/assets/1937871/10129163/18046928-65f6-11e5-9abc-24511aeafa7b.png)

**Expected:**

```
4:5 - Function 'foo' has a complexity of 2. 
```
